### DISCUSSION TITLE
Code flow discussion

### DISCUSSION THREAD

**Discussion by abhijitnathwani** [initialcommit-com/git-sim#63] [General]
Hi @initialcommit-io @paketb0te 
I was thinking what would be the best way for us, the contributors and you, to engage in code discussions, questions that we may have understanding the flow? I personally take time to get a flow or answer to the scenario why would it happen, and I believe others may have some queries that would overlap. I reckon discussing it with you would speed up that flow and help us push new features more easily. Somethings may not be bugs and our understanding of the feature may be flawed. These are not `Issues` so I don't want to keep creating Github issues for these. We can use the discussions Q&A on Github or start some discord server. 
On other topic, we can also use Github discussions to discuss features and road map for the future. `Projects` may also come in handy in the future to help track once we have enough features to work on.
Your thoughts ?

**Comment by initialcommit-io** [comment]
@abhijitnathwani Yes I completely agree.  Thanks for suggesting this.  I was looking into this before posting my last note about the 0.2.6 release, but it was getting late at night and I had never done it before so I took the lazy way out 😅 .

Let's use discussions for the things you suggested - topics to help us figure out the direction for the project and whether to implement certain features / how to implement them, questions about how the current code works, etc.

Feel free to open topics on anything you see fit.
